package.json

```diff
@@ -3,10 +3,17 @@
   "description": "Client for Contentful's Content Delivery API",
   "version": "0.0.0-determined-by-semantic-release",
   "homepage": "https://www.contentful.com/developers/documentation/content-delivery-api/",
-  "main": "./dist/contentful.node.js",
-  "browser": "./dist/contentful.browser.js",
-  "modules": "./dist/contentful.browser.js",
+  "source": "./lib/index.ts",
+  "main": "./dist/contentful.cjs.js",
+  "module": "./dist/contentful.esm.mjs",
```

Comment:
Why is this defined as `*.mjs` - should be `*.js`right?

Reply:
@marcolink Why would you want to take away the more specific `*.mjs` here, isn't this just a helping guideline for anyone or any tool using this module (which is anyways an ES module)?

Taking this thought from the parcel docs (https://parceljs.org/getting-started/library/#commonjs-and-es-modules): 
![image](https://user-images.githubusercontent.com/43542437/150136056-45bf0dfc-b2b8-42b4-9a9c-4147359b136d.png)

Reply:
But not sure, we should probably be aligned in the structure of all bundled files here 🤔

Reply:
And also just saw our internal alignment doc ...

Reply:
Okay, I saw that we aligned on always using `*.js`, so I'll change that, but just wonder what it the benefit compared to using the `cjs` and `mjs` endings?